modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -39,9 +39,13 @@ spec:
       template:
         spec:
           storageClassName: <new_storage_class> <1>
-#...
+      schedule: "0 */12 * * *" <2>
+      managedDataSource: <data_source> <3>
+# ...
 ----
 <1> Define the storage class.
+<2> Required: Schedule for the job specified in cron format.
+<3>  Required: For the custom image to be detected as an available boot source, the value of the `managedDataSource` field must match the value of the `spec.dataVolumeTemplates.spec.sourceRef.name` field in the VM template YAML file.
```

Comment:
@awels Thanks for your feedback. I might have gotten the wording from elsewhere in the docs, [Enabling automatic updates for custom boot sources, callout 4](https://docs.openshift.com/container-platform/4.15/virt/storage/virt-automatic-bootsource-updates.html#virt-autoupdate-custom-bootsource_virt-automatic-bootsource-updates). 

> For the custom image to be detected as an available boot source, the name of the image's `managedDataSource` must match the name of the template's `DataSource`, which is found under `spec.dataVolumeTemplates.spec.sourceRef.name` in the VM template YAML file.

Reply:
Like I said it is technically correct, as the two need to match, but it becomes incorrect when you have multiple VM templates referencing the same dataSourceTemplate. The VM template must match the DS template, not the other way around. And the way it is worded right now it seems like the DS template must match the VM template, which is correct most of the time but in the one case I mention it might not be correct.

Reply:
Ugh. Sorry this is all fairly new to me. [One more attempt](https://github.com/openshift/openshift-docs/pull/69051/commits/e136dc4b7ef059e8f196027edfadf7a1ab3005a4)! 

> For the custom image to be detected as an available boot source, the value of the `spec.dataVolumeTemplates.spec.sourceRef.name` parameter in the VM template must this value.